scripts/scripts.js

```diff
@@ -1424,7 +1430,35 @@ sampleRUM.always.on('convert', (data) => {
         }
       };
     }
-    console.debug('push to datalayer', evtDataLayer);
-    window.digitalData.push(evtDataLayer);
+
+    //Check if a Form Complete event has fired - and if the conversion value (data.target) or form name (data.source) is defined
+    if (evtDataLayer.event === "Form Complete" && (data.target === undefined || data.source === undefined)) {
```

Comment:
The comment and the code are contradictory, so I'm not sure if you meant:
```suggestion
    //Check if a Form Complete event has fired - and if the conversion value (data.target) or form name (data.source) is undefined
    if (evtDataLayer.event === "Form Complete" && (data.target === undefined || data.source === undefined)) {
```

or:

```suggestion
    //Check if a Form Complete event has fired - and if the conversion value (data.target) or form name (data.source) is defined
    if (evtDataLayer.event === "Form Complete" && (data.target !== undefined || data.source !== undefined)) {
```